doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1600,14 +1633,18 @@ the proof and adds it to the environment.
 Attributes
 -----------
 
-.. insertprodn all_attrs legacy_attrs
+.. insertprodn all_attrs legacy_attr
```

Comment:
You should generate these changes through `common.edit_mlg` (search for "legacy_attrs" there and do `make doc_gram_rsts`).  The insertprodn will give an error in that tool.  If you add legacy_attr to the mlg as I suggest elsewhere (good idea), you'll also need to add edits to format legacy_attr as you show it below.  A good opportunity to get your feet wet using this.

Reply:
I wanted to do this (indeed, a good idea to get used to this tool) but I was afraid that it would create merge conflicts with your own ongoing changes. WDYT?

Reply:
I suggest merging #11720 first, which applies some grammar changes from master.  Then try the tool.  Be sure to commit the orderedGrammar changes.  If you see a lot of extraneous changes in orderedGrammar and/or the rsts, let me know.  Or just push the updates and I can look at them--perhaps that's the easiest thing.

Reply:
Also commit the fullGrammar changes.  But not the others in that docgram directory.

Reply:
You also need to commit the orderedGrammar and gallina-specification-language.rst changes.  If you want Local and Global, etc. to show up in a single line in the rst, then you need to edit the mlg as I suggested or add edits in common.edit_mlg.

Reply:
Done. I backtracked on the Local / Global, etc. compacting.